server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -367,15 +365,13 @@ public TabletServer(ServerContext context) {
     this.replWorker = new ReplicationWorker(context, fs);
     this.statsKeeper = new TabletStatsKeeper();
     SimpleTimer.getInstance(aconf).schedule(() -> {
-      synchronized (onlineTablets) {
-        long now = System.currentTimeMillis();
-        for (Tablet tablet : onlineTablets.values())
-          try {
-            tablet.updateRates(now);
-          } catch (Exception ex) {
-            log.error("Error updating rates for {}", tablet.getExtent(), ex);
-          }
-      }
+      long now = System.currentTimeMillis();
```

Comment:
Will dropping the sychronized block in the simple timer cause problems?  I am not sure how much of the SimpleTimer is shared and what we should worry about.

Reply:
I don't think it will cause any problems.  It was sync before to avoid concurrent mod exceptions.  Now it will iterate over the immutable snapshot.